mono/mini/mini-runtime.h

```diff
@@ -510,11 +510,23 @@ void mono_enable_jit_map (void);
 void mono_emit_jit_map   (MonoJitInfo *jinfo);
 void mono_emit_jit_tramp (void *start, int size, const char *desc);
 gboolean mono_jit_map_is_enabled (void);
+#if defined(ENABLE_JIT_DUMP)
+#include <sys/mman.h>
```

Comment:
Is `mman.h` needed?

Reply:
That's where mmap comes from

Reply:
Yes, but it's not used in the header. Only in the `runtime.c` file.  It should go there.